components/PrintingsModal.js

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Modal, Grid } from 'semantic-ui-react';
+
+class PrintingsModal extends Component {
+  onPrintingSelect = e => {
+    e.preventDefault();
+    const cardID = e.target.id;
+    this.props.selectCard(cardID);
+  };
+  render() {
+    const { allPrintings, isOpen, onClose } = this.props;
+    return (
+      <Modal closeIcon onClose={onClose} open={isOpen}>
+        <Modal.Header>Select printing/art to display</Modal.Header>
+        <Modal.Content>
+          <Grid>
+            <Grid.Column computer="12" mobile="16">
+              {allPrintings.map(card => (
+                <h4 key={card.id}>
+                  <a id={card.id} onClick={this.onPrintingSelect} href="">
```

Comment:
Probably good to have `href="#"` - not sure if some browsers still choke on no href.